pkg/sippyserver/job_run_scan.go

```diff
@@ -0,0 +1,172 @@
+package sippyserver
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/openshift/sippy/pkg/api"
+	apijobrunscan "github.com/openshift/sippy/pkg/api/jobrunscan"
+	"github.com/openshift/sippy/pkg/db/models/jobrunscan"
+	log "github.com/sirupsen/logrus"
+)
+
+// Job run labels CRUD handlers
+
+func (s *Server) jsonListLabels(w http.ResponseWriter, req *http.Request) {
+	labels, err := apijobrunscan.ListLabels(s.db, req)
+	if err != nil {
+		failureResponse(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+	api.RespondWithJSON(http.StatusOK, w, labels)
+}
+
+func (s *Server) jsonGetLabel(w http.ResponseWriter, req *http.Request) {
+	vars := mux.Vars(req)
+	id := vars["id"]
+
+	label, err := apijobrunscan.GetLabel(s.db, id, req)
+	if err != nil {
+		failureResponse(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+	if label == nil {
+		failureResponse(w, http.StatusNotFound, "label not found")
+		return
+	}
+	api.RespondWithJSON(http.StatusOK, w, label)
+}
+
+func (s *Server) jsonCreateLabel(w http.ResponseWriter, req *http.Request) {
+	user := getUserForRequest(req)
```

Comment:
Curious what happens here and the other update/delete calls when there is no users?  It doesn't look like there is a check so if this is behind unautheticated sippy I expect it will fail since we only have read only creds but should there be a check here first that we are write enabled / have a valid user?